src/docker/build.nim

```diff
@@ -51,8 +52,8 @@ proc processGitContext(ctx: DockerInvocation) =
     raise
 
 proc processDockerFile(ctx: DockerInvocation) =
-  if ctx.dockerFileLoc == ":stdin:":
-    let input           = stdin.readAll()
+  if ctx.dockerFileLoc == chalk_common.stdin:
```

Comment:
```suggestion
  if ctx.dockerFileLoc == stdin
```

you only need the specify module name whenever there is a conflict with another module which already has `stdin` defined

speaking of that as stdlib has `stdin` lets rename to something like `stdinIndicator` or something like that to indicate its not a file-like object but an indicator that something is stdin

Reply:
I had only included it where there was said conflict. I'll swap the name around.